bundler/lib/bundler/plugin.rb

```diff
@@ -113,6 +113,9 @@ def gemfile_install(gemfile = nil, &inline)
         return if definition.dependencies.empty?
 
         plugins = definition.dependencies.map(&:name).reject {|p| index.installed? p }
+
+        return if plugins.empty?
```

Comment:
If I have

```
plugin 'extended_bundler-errors', '0.3.2'
```

in my Gemfile, and I change it to

```
plugin 'extended_bundler-errors', '0.3.3'
```

I think this is going to change things to not install the new version?

Reply:
Yeah, you're right, I'll fix it.